ReactAndroid/build.gradle

```diff
@@ -330,7 +330,9 @@ dependencies {
     api 'com.google.code.findbugs:jsr305:3.0.2'
     api "com.squareup.okhttp3:okhttp:${OKHTTP_VERSION}"
     api "com.squareup.okhttp3:okhttp-urlconnection:${OKHTTP_VERSION}"
-    api 'com.squareup.okio:okio:1.14.0'
+    api 'org.bouncycastle:bcprov-jdk15on:1.60'
```

Comment:
Why not just use `okhttp-tls`.

Reply:
I looked into https://github.com/square/okhttp/blob/master/CHANGELOG.md and added those lines